src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -190,6 +190,8 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
         options['prediction_window'] = prediction_window
         options['batch_size'] = batch_size
         options['max_iterations'] = max_iterations
+        options['verbose'] = verbose
+        options['num_examples'] = len(dataset)
```

Comment:
I don't think it makes sense the pass `num_examples` explicitly. Isn't this implicit in `dataset`? The C++ side can record this field itself from `dataset`.